tests/main/interfaces-cups/test-snapd-consumer/bin/sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+PS1='$ '
```

Comment:
what's this for here? just to aid in debugging?

Reply:
Many of the `tests/main/interfaces-*` tests have this and I just copied it over.

Reply:
This prevents a bug in sh, which cannot cope with PS1 set by spread, causing an infinite loop. It allows running `snap run test-snapd-consumer.sh` without an error

Reply:
@zyga should this be in every snap that runs `/bin/sh` as a snap app?